template.pkr.hcl

```diff
@@ -95,15 +125,15 @@ build {
     execute_command = "echo 'packer' | sudo -S env {{ .Vars }} {{ .Path }}"
   }
 
-
   provisioner "shell" {
     environment_vars = ["DEBIAN_FRONTEND=noninteractive", "LC_ALL=C", "LANG=en_US.UTF-8", "LC_CTYPE=en_US.UTF-8"]
     scripts          = ["files/scripts/999-cleanup.sh"]
     execute_command  = "echo 'packer' | sudo -S env {{ .Vars }} {{ .Path }}"
   }
 
   # Store details in packer-manifest.json
-  # to retrieve AMI
+  # to retrieve AMI and GCE image IDs
   post-processor "manifest" {
```

Comment:
should we remove this? do we need this information anywhere?

Reply:
Ah yeah this is important so that I can "publish" the image IDs to the marketplace. There are two steps:

1. Build (packer will build and add the image to GCP)
2. Publish (Associate the image with a new "version" of the item in marketplace.

You can see it being referenced here:
https://github.com/coder/packages/blob/7d47b04ce0ce9c3c110136fed6465c06eb8b37c0/.github/workflows/aws/submit_ami.sh#L3-L5